src/briefcase/commands/run.py

```diff
@@ -159,15 +159,53 @@ def add_options(self, parser):
             "--update",
             action="store_const",
             const=True,
-            help="Update and rebuild the app before execution",
+            help="Update and rebuild the app before running",
         )
         parser.add_argument(
             "--no-update",
             dest="update",
             action="store_const",
             const=False,
-            help="Prevent any automated update and rebuild before running.",
+            help="Prevent any automated update of requirements before running.",
```

Comment:
"of requirements" should be removed.

More generally, since several options are now shared between the update, build and run commands, could they be factored out? The only difference between the three copies is the last few words of the help text, which is either "before building", "before running", or nothing at all. But think those words are unnecessary, since the user already knows what command they're looking at. And removing them would allow each argument description to fit on a single line, making the help text easier to read.

Reply:
Yeah - I agree there's lots of duplication; I was hesitant to pull out a common definition because there's no obvious "common" base class; however, I guess adding a utility method to the base command that can be invoked as needed is an option. I've made this change.

I'm not sure I agree about truncating the help text, though. It never hurts to be explicit, and help text is the very definition of "you're only looking at it if you don't know what you're doing", so assuming that the context is obvious is problematic. I'm not overly worried about running onto multiple lines, either - that only gets enforced if you're literally at 80 characters. From the look of it, if your terminal is only slightly wider, the wrapping stops; and there's much bigger formatting problems caused by the length of `--no-update-requirements` (and similar) options).